Makefile

```diff
@@ -137,3 +137,8 @@ vendor-update:
 .PHONY: test-jsonschema-generation
 test-jsonschema-generation:
 	docker run -v `pwd`/pkg/spec/types.go:/data/types.go:ro,Z surajd/kedgeschema
+
+# Update schema which is used for json-schema validation
+.PHONY: update-schema
+update-schema:
+	docker run --rm  -v `pwd`:/data:Z  surajnarwade/update-kedge-schema
```

Comment:
There is kedge org in docker hub, we should use that.

Reply:
it seems that I don't have access to that org, can someone please add me there

Reply:
same here :-( @cdrage should have access.